src/client.py

```diff
@@ -0,0 +1,27 @@
+"""Create a client to interact with a server at localhost:5353."""
+
+import socket
+
+
+def client(message):
+    """Send message to server and get reply."""
+    header = 'GET /index.html HTTP/1.1\r\nHost: www.example.com\r\n\r\n'
+    send_msg = header + message + '\r\n\r\n'
```

Comment:
nope, don't do this.  The header should not just be attached here automatically.